iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/tools/ConversionToolUtils.java

```diff
@@ -100,6 +102,23 @@ public static Properties readProperties(CommandLine line, OptionType optionType,
         return properties;
     }
 
+    public static Option createSkipPostProcOption() {
+        return Option.builder().longOpt(SKIP_POSTPROC)
+                .desc("skip network importer post processors (when configured)")
+                .build();
+    }
+
+    public static ImportConfig createImportConfig(CommandLine line) {
+        return createImportConfig(line, ImportConfig.load());
+    }
+
+    public static ImportConfig createImportConfig(CommandLine line, ImportConfig importConfig) {
+        if (line.hasOption(SKIP_POSTPROC)) {
+            importConfig.setPostProcessors();
```

Comment:
readable? maybe its just my problem

Reply:
Okay, I added a `clearPostProcessors()` method (more readable) in `ImportConfig` and used it here